demo/api/src/products/entities/product-variant.entity.ts

```diff
@@ -33,7 +34,7 @@ export class ProductVariant extends BaseEntity<ProductVariant, "id"> implements
         // search: true, // not yet supported for nested
         // filter: true, // not yet supported for nested
         // sort: true, // not yet supported for nested
-        // input: true, // not supported for nested, doesn't make sense
+        input: false, // default is true, but usually not needed if it's a rootArg
```

Comment:
Is this necessary, or is this removed anyway with this change? 8546e99f086aeb25de05eb463b759a232746fdac